src/Backupper/AbstractBackupper.php

```diff
@@ -96,6 +114,28 @@ public function isVerbose(): bool
         return $this->verbose;
     }
 
+    /**
+     * Provide the built-in default options that will be used if none is given
+     * through the dedicated constructor argument.
+     */
+    protected function getBuiltinDefaultOptions(): string
+    {
+        return '';
```

Comment:
I don't know if this method should not be abstract to force people to think about it when they add a new backupper ?

Reply:
I chose to avoid having to implement it for "nothing", but "force people to think about it when they add a new backupper" seems not absurd :wink: I can change that if you prefer that we do it this way.

Reply:
I have no strong opinion, if was just a suggestion, as you like!

Reply:
I keep the current version. This can be easily changed later.